compiler/runtime/CMakeLists.txt

```diff
@@ -27,4 +27,5 @@ compiler_library(runtime
 	${CMAKE_CURRENT_LIST_DIR}/OMRCodeCacheManager.cpp
 	${CMAKE_CURRENT_LIST_DIR}/OMRCodeCacheMemorySegment.cpp
 	${CMAKE_CURRENT_LIST_DIR}/OMRCodeCacheConfig.cpp
+	${CMAKE_CURRENT_LIST_DIR}/OMRRSSReport.cpp
```

Comment:
Shouldn't an equivalent change be made in makefiles like `jitbuilder/build/files/common.mk` for example?

Reply:
I suspect this is the reason internal builds are failing (e.g. z/OS):
```
 IEW2456E 9207 SYMBOL addRSSItem__Q2_3OMR9RSSRegionFPQ2_3OMR7RSSItemiPCc        
          UNRESOLVED.  MEMBER COULD NOT BE INCLUDED FROM THE DESIGNATED CALL    
          LIBRARY.                                                              
 IEW2456E 9207 SYMBOL _instance__Q2_3OMR9RSSReport UNRESOLVED.  MEMBER COULD NOT
          BE INCLUDED FROM THE DESIGNATED CALL LIBRARY. NAME SPACE = 3          
FSUM3065 The LINKEDIT step ended with return code 8.
/j9tmp/bld_74387/bld_zos_390/compiler/../compiler/build/rules/zos-xlc/common.mk:47: recipe for target '/j9tmp/bld_74387/bld_zos_390/compiler/../libj9jit29.so' failed
```

Reply:
@gita-omr ^